readme.md

```diff
@@ -149,6 +149,7 @@
 - [autodoc](https://github.com/context-labs/autodoc) - Automatically generate codebase documentation.
 - [GPT3 WordPress Post Generator](https://github.com/nicolaballotta/gpt3-wordpress-post-generator) - Generate WordPress blog posts.
 - [aiac](https://github.com/gofireflyio/aiac) - Infrastructure-as-Code generator.
+- [tenere](https://github.com/pythops/tenere) - TUI interface for LLMs written in Rust.
```

Comment:
You are assuming everyone knows what TUI is. I had to Google it.

Also:

> Only ChatGPT is supported for the moment. But I'm planning to support more models in the future.

The description should reflect the current state of the project, not future plans.

Reply:
what about this description instead
```
Terminal interface (TUI) for ChatGPT written in Rust.
```
looks good ?

Reply:
👍

Reply:
updated.